pyproject.toml

```diff
@@ -26,13 +26,14 @@ pytest = ">=6.1.2"
 pytest-watch = ">=4.2.0"
 moto = "^2.2.11"
 pandas = "^1.5.0"
+duckdb-engine = "0.7.0"
```

Comment:
any reason to pin to this exact version rather than `"^0.7.0"`?

Reply:
updated